scope.c

```diff
@@ -1846,6 +1846,63 @@ Perl_cx_dump(pTHX_ PERL_CONTEXT *cx)
 #endif	/* DEBUGGING */
 }
 
+void
+Perl_create_sv_destructor(pTHX_ SV *subref, SV *args) {
+    PERL_ARGS_ASSERT_CREATE_SV_DESTRUCTOR;
+    SV *variable = newSV_type_mortal(SVt_IV);
+    MAGIC* magic = sv_magicext(variable, subref, PERL_MAGIC_destruct,
+                               &PL_vtbl_destruct, (char *)args, args ? HEf_SVKEY : 0);
+    magic->mg_flags |= MGf_LOCAL;
```

Comment:
The MGf_LOCAL line isn't needed if no localization handler is set, and no localization handler is needed if «value_magic => 1» is set in `mg_vtable.pl` (and then `MAGIC* magic`  doesn't have to be saved either).

Reply:
Done.